### PR TITLE
Add old Configuration System Binplacing doc

### PR DESCRIPTION
**Pull request by ViktorHofer**

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Part of the TargetFramework SDK is the binplacing infrastructure which is described below. The TargetFramework SDK infrastructure itself isn't yet documented.
+
+## BinPlacing
+
+Supports copying to additional paths based on which configuration among `BuildConfigurations` is best for a given `BinplaceConfiguration`
```
Comment (Anipik) [inline src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md:5]: BinplaceConfiguration -> BinplaceTargetFramework

Comment (Anipik) [inline src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md:5]: BuildConfigurations -> targetFrameworks
Configuration -> targetframework

File: src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### BinPlacing Items
+
+- BinPlaceItem
+    - Typically computed by the BinPlacing targets to determine what assets to binplace.
+    - Identity: source of file to binplace.  For example: the built output dll, pdb, content files, etc.
+    - Metadata:
+        - TargetPath: when specified can indicate the relative path, including filename, to place the item.
+
+- BinPlaceConfiguration
```
Comment (Anipik) [inline src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md:15]: BinplaceConfiguration -> BinplaceTargetFramework

File: src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - Typically specified by the repository to control the output directories of projects.
+    - Identity: configuration to binplace for.  A binplace configuration is *active* if it is the best configuration for the currently building project configuration.  When active it's behavior is defined by the meatadata below.
```
Comment (Anipik) [inline src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/README.md:17]: configuration -> targetFramework

**Comment by ViktorHofer** [comment]
Thanks, updated the doc.

**Pull request merged**